suites/pacific/cephadm/tier-1_skip_dashboard.yaml

```diff
@@ -246,3 +246,39 @@ tests:
           password: admin@123
       destroy-cluster: false
       abort-on-fail: true
+  - test:
```

Comment:
Could we write a separate suite holding the below tests? I noticed that the test suite name is `skip-dashboard` but we are testing some of the dashboard features.

Reply:
By default, all monitoring stack will be enabled, In the skip-dashboard only we need to enable them manually 
Hence these tests should come under this suit itself.